### DISCUSSION TITLE
Badge of Justice/Points from Hardmode, etc. as a currency for T3

### DISCUSSION THREAD

**Discussion by andyxtoday** [chromiecraft/chromiecraft#5548] [Suggestions]
Some people, like me, don't particularly enjoy leveling for the sake of leveling and try to focus all energy into one main character, with maybe one or two alts. And from my perspective, that is essentially what the challenge modes are, leveling for the sake of leveling, but more tedious. Now I don't want minimize the effort people are putting in to going through those challenges, more power to them, it's just not for me (and many others I would think.) 

So I'm suggesting an alternative way of reaching _some_ if the rewards from the challenge modes, **the OG Naxx sets**, as a long-term goal and another reason to continue farming Heroics after buying BiS pieces. Maybe even the weapons? Something that can be discussed, no doubt. I haven't really thought too hard on what each item should cost, but that is also something that can be discussed.

I also know that a lot of the PvP'er crowd have discussed having T3/Naxx weapons as a PvP rewards as well, which I think would be great. But I don't really partake in PvP, so I'm leaving it up to them to make a proper suggestion about that.

So yeah, I hope this is something that resonate with people!

**Comment by trinitarius** [comment]
I agree with T3 having multiple ways of being acquired. 

I can get behind the idea of getting it as a reward from PvE or PvP means, maybe by the Hardmode points, Badges of Justice, Honor or BG marks but the cost of the T3 items should be an **equivalent of the time investment** of challenge modes. If getting T3 tmog by Badges of Justice or any other means is way faster than doing all three challenge modes it will result in players mostly doing the most efficient way than the most preferred way. Which will result in one of the acquiring methods to be the most popular.

Balancing it is probably quite impossible but I would welcome it to be a proper time effort. Doing all three challenge modes separately or together is a serious grind and time effort and it would be important to also be of the same difficulty to preserve the value of T3 tmog.

For the weapons I am unsure, since it might be that they are the exclusive additional reward for doing all three challenge modes together - which we don't know yet. If that would be the case and weapons will be also acquirable by other means then there is no exclusive reward for the hardest/grindiest way of doing the challenges (doing 3/3) left which will result in no one doing those anymore and some people have invested days of playtime already into this specific hard way of doing the challenges. So we might have to wait to see if there are any additional exclusive rewards for 3/3 and if there are any, they must stay exclusive which would be unfortunate if it is Naxx weapons. Otherwise exclusive rewards for 3/3 would need to be added to the Naxx weapons or changed altogether so that there is still a reason left to do 3/3.

My two cents.

**Comment by Apathyxia** [comment]
Shortly, I think the implemented Hardmode score could provide an additional source for T3 rather than badges.

**Reply by andyxtoday** [reply]
Yes, I really like this idea!

**Comment by 55Honey** [comment]
@andyxtoday @LigmoBallsins let's keep GitHub on topic and stay clear of personal attacks.

There are no plans to provide additional sources for T3 sets currently.

Thanks for the suggestions.
